chapters/publication.tex

```diff
@@ -1,96 +1,171 @@
-so it is easy to start up a project
-and replicate a lot of the underlying setup code.
-One  of the most common issues you will face while using Overleaf will be special characters, namely
-\texttt{\&}, \texttt{\%}, and \texttt{\_},
-which need to be \textbf{escaped} (instructed to interpret literally)
-by writing a backslash (\texttt{\textbackslash}) before them,
-such as  \texttt{40\textbackslash\%} for the percent sign to function.
-Another issue is that you need to upload input files
-(such as figures and tables) manually.
-This can create conflicts when these inputs are still being updated --
-namely, the main document not having the latest results.
-One solution is to move to Overleaf only once there will not be substantive changes to results.
-
-Other popular desktop-based tools for writing \LaTeX are TeXstudio\sidenote{\url{https://www.texstudio.org}} and atom-latex\sidenote{\url{https://atom.io/packages/atom-latex}}.
-They allow more advanced integration with Git,
-among other advantages, but the entire team needs to be comfortable
+who simply want to get to writing, like senior researchers.
+While integrated editing and compiling tools like TeXStudio\sidenote{
+  \url{https://www.texstudio.org}}
+and \texttt{atom-latex}\sidenote{
+  \url{https://atom.io/packages/atom-latex}}
+offer the most flexibility to work with \LaTeX\ on your computer,
+such as advanced integration with Git,
+the entire group of writers needs to be comfortable
 with \LaTeX\ before adopting one of these tools.
+They can require a lot of troubleshooting at a basic level at first,
+and non-technical staff may not be willing or able to acquire the required knowledge.
+Therefore, to take advantage of the features of \LaTeX,
+while making it easy and accessible to the entire writing team,
+we need to abstract away from the technical details where possible.
 
-One of the important tools available in \LaTeX\ is the \textbf{BibTeX bibliography manager}.\sidenote{\url{http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.365.3194&rep=rep1&type=pdf}}
-This tool stores unformatted references
-in an accompanying \texttt{.bib} file,
-and \LaTeX\ then inserts them in text
-using the \texttt{\textbackslash cite\{\}} command.
-With this structure, \LaTeX\ automatically pulls
-all the citations into text. \LaTeX\ allows you to specify
-how they should be displayed in text
-(ie, as superscripts, inline references, etc.)
-and how the bibliography should be styled and in what order.
-A reference in the \texttt{.bib} file
-can be copied directly from Google Scholar
-by clicking "BibTeX" at the bottom of the Cite window.
+One of the most important tools available in \LaTeX\ is the BibTeX bibliography manager.\sidenote{
+  \url{http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.365.3194&rep=rep1&type=pdf}}
+BibTeX keeps all the references you might use in an auxiliary file,
+then references them as plain text in the document using a \LaTeX\ command.
```

Comment:
I do not understand the usage of _plain text_ in this context

Reply:
768c65dbd5782523bcf5ab8a1864d08424e25d36